Python/pywarpx/Bucket.py

```diff
@@ -73,11 +75,11 @@ def attrlist(self):
                     continue
                 # --- For lists, tuples, and arrays make a space delimited string of the values.
                 # --- The lambda is needed in case this is a list of strings.
-                rhs = ' '.join(map(lambda s : repr(s).strip("'"), value))
+                rhs = ' '.join(map(lambda s : str(s).strip("'"), value))
```

Comment:
```suggestion
                rhs = ' '.join(map(lambda s : f'{s}', value))
```
Note that with the f-string, the strip is not needed.